src/lib.rs

```diff
@@ -17,87 +21,189 @@ impl Default for Syntax {
     }
 }
 
+/// A field frequency
```

Comment:
Worth mentioning this entity is called "label" in `descriptor.proto`.

Reply:
Indeed.

I also found that they use the term *Rule* [here](https://developers.google.com/protocol-buffers/docs/proto3#specifying-field-rules) and [here](https://developers.google.com/protocol-buffers/docs/proto#specifying-field-rules). Note that the `singular` rule has not been implemented (defaulting to `optional`) but I can add it if needed.

I tend to prefer *Rule* over *Label* but I am happy to change if you think this is necessary.